src/Parser/VengeanceDemonHunter/CPM_ABILITIES.js

```diff
@@ -0,0 +1,71 @@
+import SPELLS from 'common/SPELLS';
+
+export const SPELL_CATEGORY = {
+  ROTATIONAL: 'Spell',
+  COOLDOWNS: 'Cooldown',
+};
+
+const CPM_ABILITIES = [
+  {
+    spell: SPELLS.SPIRIT_BOMB_TALENT,
+    isActive: combatant => combatant.lv90Talent === SPELLS.SPIRIT_BOMB_TALENT.id,
```

Comment:
Tip: you can use `hasTalent` for checking for talents, it's a little bit easier to use.